src/ru/nsu/ccfit/zuev/osu/game/cursor/flashlight/MainFlashLightSprite.java

```diff
@@ -0,0 +1,43 @@
+package ru.nsu.ccfit.zuev.osu.game.cursor.flashlight;
+
+import org.anddev.andengine.entity.modifier.ScaleModifier;
+import org.anddev.andengine.opengl.texture.region.TextureRegion;
+
+
+public class MainFlashLightSprite extends FlashlightAreaSizedSprite {
+    public static final float BASE_SIZE = 8f;
+    public static final int TEXTURE_WIDTH = 1024;
+    public static final int TEXTURE_HEIGHT = TEXTURE_WIDTH / 2;
+    public final float AREA_SHRINK_FADE_DURATION = 0.8f;
```

Comment:
Would suggest something like `AREA_CHANGE_FADE_DURATION` since this is also used to expand the area.